src/page-templates/content-page/content-page-template.tsx

```diff
@@ -632,11 +639,18 @@ const ContentPageTemplate: NextPage<ContentPageProps> = ({
                 openGraph={{
                     url: seo?.og_url,
                     title: seo?.og_title,
-                    type: seo?.og_type,
+                    type: ogType,
                     description: seo?.og_description,
                     images: seo?.og_image?.url
                         ? [{ url: seo?.og_image?.url }]
                         : [],
+                    article: {
+                        publishedTime: Array.isArray(contentDate)
+                            ? undefined
+                            : contentDate,
+                        modifiedTime: updateDate,
+                        authors: authors.map(({ name }) => name),
```

Comment:
What should the content of authors field be? I have come across different examples:

1. The example in the [documentation](https://github.com/garmeeh/next-seo) is a list of URLs.
2. LinkedIn Open graph shows a [profile array](https://www.linkedin.com/pulse/open-graph-protocol-atoconn-india/)

Reply:
Ah good call, I just saw an `string[]` in the documentation and assumed it would be names. I can replace that with a URL pointing to the author page.

Reply:
I do note that I still don't know what are the correct expected values for this field, but perhaps going with the documentation is the safest route for now.